Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,10 +101,54 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
+            var newArray = ProcessArray(array, index);
+            return newArray;
+        }
+
+        private static int[] ProcessArray(int[] array, int index)
```

Comment:
You can write this in RemoveAt. It's not necessary to put this in a new function. Also ProcessArray is very generic so RemoveAt will sound more of what is happening than ProcessArray.

Reply:
done